src/patternfly/components/Card/card.scss

```
@@ -3,6 +3,9 @@
   --pf-c-card--BackgroundColor: var(--pf-global--BackgroundColor--100);
   --pf-c-card--BoxShadow: var(--pf-global--BoxShadow--sm);
   --pf-c-card--m-hoverable--hover--BoxShadow: var(--pf-global--BoxShadow--lg);
+  --pf-c-card--m-selectable--hover--BoxShadow: var(--pf-global--BoxShadow--lg);
+  --pf-c-card--m-selectable--m-selected--before--Height: var(--pf-global--BorderWidth--lg);
+  --pf-c-card--m-selectable--m-selected--before--BackgroundColor: var(--pf-global--link--Color);
```

I think we should reserve `link--Color` for links. We use `--pf-global--active-color--100` in the other components to indicate some active thing like tabs current state or the data-list/table expanded/selected state. I think an exception to that is nav link current border and I wonder if we should update that, too, or if it should always match the link color of the nav item. I'll make a note to ask @mceledonia about that when he gets back.

```suggestion
  --pf-c-card--m-selectable--m-selected--before--BackgroundColor: var(--pf-global--active-color--100);
```